### ISSUE TITLE
Datagrid: Cannot select all using keyboard when list is multi-select/mixed

### ISSUE THREAD

**Issue by krishollenbeck** [infor-design/enterprise#3921]
**Describe the bug**
Cannot select all rows in datagrid when using keyboard navigation. This appears to only be happening in mixed select scenario.

**To Reproduce**

USE CASE 1:
1. Go to http://localhost:4000/components/datagrid/example-mixed-selection.html
2. Tab to first row using keyboard navigation.
3. Press "Ctrl + A"

USE CASE 2:
1. Go to http://localhost:4000/components/datagrid/example-mixed-selection.html
2. Tab to first row using keyboard navigation.
3. Press "Arrow Up" key. Press enter or space to select all.

**Expected behavior**
All rows should be selected.

**Version**
4.29.0-dev

**Screenshots**
![ctrl+a-notworking](https://user-images.githubusercontent.com/1056684/81972778-baf82f80-95e8-11ea-9f34-1900f7478bfb.gif)

**Platform**
 - OS Version: Windows 10
 - Browser Name: Chrome
 - Browser Version: 81.0.4044.138 (Official Build) (64-bit)

**Comment by janahintal** [comment]
QA Failed. Unable to test in IE as the grid didn't load. 
https://master-enterprise.demo.design.infor.com/components/datagrid/example-mixed-selection.html
![image](https://user-images.githubusercontent.com/45757016/84134215-d5ec8100-aa7a-11ea-95be-72de6b08d3a6.png)

**Comment by tmcconechy** [comment]
@janahintal @CindyMercadoReyes 

Can you test this again? I went to this page in Windows 10 IE11 and had no problems:

https://master-enterprise.demo.design.infor.com/components/datagrid/example-mixed-selection.html
